### PR TITLE
Chapter 7: Sign up

### PR DESCRIPTION
**Pull request by tungts1101**
image

### CODE REVIEW

**Comment by tungts1101** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+  end
+
+  def show
+    @user = User.find params[:id]
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Welcome to the Sample App"
```
Comment (hungnh103) [inline app/controllers/users_controller.rb]: I18n

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<% provide :title, "Sign up" %>
```
Comment (hungnh103) [inline app/views/users/new.html.erb]: I18n

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<h1>Sign up</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
```
Comment (hungnh103) [inline app/views/users/new.html.erb]: convention

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control" %>
+
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+
+      <%= f.label :password_confirmation, "Confirmation" %>
```
Comment (hungnh103) [inline app/views/users/new.html.erb]: I18n

**Comment by tungts1101** [comment]
ready

**Pull request merged**